matrix_sdk/src/client.rs

```diff
@@ -2686,6 +2686,12 @@ impl Client {
 
         Ok(())
     }
+
+    /// Gets information about the owner of a given access token.
+    pub async fn whoami(&self) -> Result<whoami::Response> {
+        let request = whoami::Request::new();
+        self.send(request, None).await
```

Comment:
Can't this just be
```suggestion
        self.send(whoami::Request::new(), None).await
```
?

Reply:
Other methods in the `Client` (like `join_room_by_id`) do the two-liner, so I adapted. Personally I also prefer giving distinct things explicit variables and names, makes things easier to read (and code is read more often than written)